modules/hostap/CMakeLists.txt

```diff
@@ -209,11 +210,31 @@ zephyr_library_compile_definitions_ifdef(CONFIG_WIFI_NM_WPA_SUPPLICANT_AP
 	CONFIG_NO_VLAN
 	CONFIG_NO_ACCOUNTING
 	NEED_AP_MLME
+	CONFIG_IEEE80211AC
```

Comment:
why is this needed? As this is independent to how the AP is being setup, it should be in a separate commit.

Reply:
Our chip supports 11AC(both STA and AP) and is enabled by default. Without this flag, 11AC related source code won't be compiled and set conf->ieee80211ac is useless.

Reply:
ok, can you please put this in a separate commit.

Reply:
Updated.